### ISSUE TITLE
 Is data_xml_process_Baseline_withlabel.py missed?

### ISSUE THREAD

**Issue by ccmCCMfk** [hjlllll/Functionalgrasp#4]
Hi,thanks for your cool work,I got an error when running write_visualize.py in grasptype_bruch folder,"from data_xml_process_Baseline_withlabel import RTJlabel,GraspNetCollate,No mudule Found.Is data_xml_process_Baseline_withlabel.py missed?

**Comment by hjlllll** [comment]
uploaded

**Comment by ysyf293** [comment]
> Hi,thanks for your cool work,I got an error when running write_visualize.py in grasptype_bruch folder,"from data_xml_process_Baseline_withlabel import RTJlabel,GraspNetCollate,No mudule Found.Is data_xml_process_Baseline_withlabel.py missed?

When I execute this file, Load the dataset dataset_hjl/labeled_data/use/about_grasp/0.002_s420_465-100_ex-True_seg-True_train_record-is111111 downloaded from grasp_brunch -new_withlabelgrasp-expanded-100-unsampled.pkl displays an error in for i, batch in enumerate(train_dataloader) : 'list' object cannot be interpreted as an integer. I don't know what went wrong. Have you ever met this situation?

**Comment by ccmCCMfk** [comment]
I just tried it a few minutes ago. I have the same problem as you. I haven't found the cause yet.Maybe there is something wrong with "collate_fn=GraspNetCollate" in “train_dataloader = DataLoader(RTJlabel(), num_workers=8, batch_size=1,shuffle=False, drop_last=False, collate_fn=GraspNetCollate)”,I ignored this part before since it didn't affect the final result.

**Comment by hjlllll** [comment]
hey, this file is wrong, we did not intend to upload it, if want to visualize mapped grasp poses, use map_to_shadow.py. I have deleted this wrong file, sorry for that

**Comment by ccmCCMfk** [comment]
Okay,thank you
